docs/articles/developer-info/NUnit-Copyright-Notice.md

```diff
@@ -1,5 +1,7 @@
 # NUnit Copyright Notice
 
+## NUnit
+
```

Comment:
As far as I know, `Copyright (c) yyyy Charlie Poole` is not valid for any active project. However, since this is not a change to what was already out there I'm OK with merging it.

Reply:
@CharliePoole I added a commit to this PR that changes the copyright to match that in the other projects. It looks like we were both reviewing at the same time 😄